docker/test/integration/test_rdkafka.py

```diff
@@ -74,3 +74,15 @@ def stop_kafka():
         cluster.rm_out_child('/failure')
         assert cluster.check_output(30, dir='/failure')
 
+def test_ssl():
+    """
+    Verify security connection
+    """
+    producer_flow = GetFile('/tmp/input') >> PublishKafkaSSL() >> ('failure', LogAttribute())
+
+    with DockerTestCluster(KafkaValidator('test')) as cluster:
```

Comment:
What does KafkaValidator do?

Reply:
It first finds a consumer container, tries to read the output file from that container and validates its content.
I did not think it through at the first place. We can get rid of it and use SingleFileOutputValidator with the help of volume.

Reply:
Just realized that I already did sth similar in test_broker_on_off without noticing.